SIAB/spillage/api.py

```
@@ -1,5 +1,21 @@
 """this file defines interface between newly implemented spillage optimization algorithm
 with the driver of SIAB"""
+import os
+import re
+from copy import deepcopy
+import numpy as np
+import matplotlib.pyplot as plt
+from SIAB.spillage.orbio import read_param, write_nao, write_param
+from SIAB.spillage.spillage import Spillage_jy, Spillage_pw, flatten,\
+    initgen_jy, initgen_pw, flatten
```

Ruff

Redefinition of unused `flatten` from line 9

Remove definition: `flatten`

(F811)